lib/src/Uncertainty/Algorithm/MetaModel/MetaModelAlgorithm.cxx

```diff
@@ -54,15 +54,38 @@ MetaModelAlgorithm::MetaModelAlgorithm(const Sample & inputSample,
 
 /* Constructor with parameters */
 MetaModelAlgorithm::MetaModelAlgorithm(const Sample & inputSample,
+                                       const Point & weights,
+                                       const Sample & outputSample)
+  : MetaModelAlgorithm(inputSample, weights, outputSample, UserDefined(inputSample))
+{
+  // Nothing to do
+}
+
+/* Constructor with parameters */
+MetaModelAlgorithm::MetaModelAlgorithm(const Sample & inputSample,
+                                       const Sample & outputSample,
+                                       const Distribution & distribution,
+                                       const Function & model)
```

Comment:
I would rather avoid that during the review process, because it makes so that the comments and suggestions are lost. This makes impossible to use the "Commit suggestion" and "Add suggestion to batch" features of github. Squashing can be done in the end, when the code is OK and the PR is finally approved. In the meantime, it hides the changes that are done when each review is done. This practice of squashing the commit during the review may imply that the author has something to hide, which, I suppose, is not your motivation.

Reply:
Ouch! I read your comment too late! Sorry for that.